### PR TITLE
Add and Review Logs in Connector

### PR DESCRIPTION
**Pull request by ravjotbrar**
### Summary

Added and reviewed current logs in connector. 

### Description

Logs added include sql statements and high level indicators to let user know which major event is taking place (writing to parquet, copy to vertica, export from vertica, etc.)

### Related Issue

https://github.com/vertica/spark-connector/issues/197

### Additional Reviewers

@jonathanl-bq 
@alexr-bq

### CODE REVIEW

File: connector/src/main/scala/com/vertica/spark/datasource/core/VerticaDistributedFilesystemReadPipe.scala [hunk 1]
```diff
@@ -210,6 +211,7 @@ class VerticaDistributedFilesystemReadPipe(
     val delimiter = if(fileStoreConfig.address.takeRight(1) == "/" || fileStoreConfig.address.takeRight(1) == "\\") "" else "/"
     val hdfsPath = fileStoreConfig.address + delimiter + config.tableSource.identifier
     logger.debug("Export path: " + hdfsPath)
+    logger.info("Reading data from Parquet file.")
```
Comment (alexr-bq) [inline connector/src/main/scala/com/vertica/spark/datasource/core/VerticaDistributedFilesystemReadPipe.scala:214]: Don't think this is the right place to log this message, before those files have been exported.

File: connector/src/main/scala/com/vertica/spark/datasource/core/VerticaDistributedFilesystemWritePipe.scala [hunk 2]
```diff
@@ -156,10 +157,14 @@ class VerticaDistributedFilesystemWritePipe(val config: DistributedFilesystemWri
 
   def buildCopyStatement(targetTable: String, columnList: String, url: String, rejectsTableName: String, fileFormat: String): String = {
     if (config.mergeKey.isDefined) {
-      s"COPY $targetTable FROM '$url' ON ANY NODE $fileFormat REJECTED DATA AS TABLE $rejectsTableName NO COMMIT"
+      val copyStatement = s"COPY $targetTable FROM '$url' ON ANY NODE $fileFormat REJECTED DATA AS TABLE $rejectsTableName NO COMMIT"
+      logger.info("The copy statement is: \n" + copyStatement)
```
Comment (alexr-bq) [inline connector/src/main/scala/com/vertica/spark/datasource/core/VerticaDistributedFilesystemWritePipe.scala:161]: Would rather see this log statement outside the function

**Review by alexr-bq — APPROVED** [review]

File: connector/src/main/scala/com/vertica/spark/datasource/core/VerticaDistributedFilesystemReadPipe.scala (hunk 1 above)
Comment (ravjotbrar) [reply connector/src/main/scala/com/vertica/spark/datasource/core/VerticaDistributedFilesystemReadPipe.scala:214]: Good point. I'll move it to the bottom of this method.

File: connector/src/main/scala/com/vertica/spark/datasource/core/VerticaDistributedFilesystemWritePipe.scala (hunk 2 above)
Comment (ravjotbrar) [reply connector/src/main/scala/com/vertica/spark/datasource/core/VerticaDistributedFilesystemWritePipe.scala:161]: I agree

**Review by jonathanl-bq — APPROVED** [review]

**Pull request merged**